apps/src/music/player/ToneJSPlayer.ts

```diff
@@ -169,6 +171,13 @@ class ToneJSPlayer implements AudioPlayer {
       return;
     }
 
+    if (this.currentPreview?.url !== sample.sampleUrl) {
+      console.log(
```

Comment:
Do we need this on production? I've noticed that Music Lab already logs a lot to the console and I wonder how useful it is, outside of us confirming things are working.

Reply:
I had the same question.  It has been quite useful for making sure it's working, but agreed that we want to keep the console relatively quiet. A compromise would be to leave it in for a week or two, just to help analyze if it works well on a variety of environments, but we can remove it after that.

Reply:
Works for me!